app/src/main/java/com/connectycube/messenger/CreateChatDialogActivity.kt

```diff
@@ -0,0 +1,147 @@
+package com.connectycube.messenger
+
+import android.content.Intent
+import android.os.Build
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.view.Window
+import android.widget.ProgressBar
+import android.widget.Toast
+import androidx.appcompat.app.ActionBar.DISPLAY_HOME_AS_UP
+import androidx.appcompat.app.ActionBar.DISPLAY_SHOW_TITLE
+import androidx.lifecycle.ViewModelProviders
+import androidx.lifecycle.get
+import androidx.lifecycle.observe
+import androidx.recyclerview.widget.DefaultItemAnimator
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.connectycube.chat.model.ConnectycubeChatDialog
+import com.connectycube.messenger.adapters.CheckableUsersAdapter
+import com.connectycube.messenger.utilities.InjectorUtils
+import com.connectycube.messenger.viewmodels.CreateChatDialogViewModel
+import com.connectycube.messenger.vo.Status
+import com.connectycube.users.model.ConnectycubeUser
+
+class CreateChatDialogActivity : BaseChatActivity(), CheckableUsersAdapter.CheckableUsersAdapterCallback {
+
+    private lateinit var createChatDialogViewModel: CreateChatDialogViewModel
+    private lateinit var usersAdapter: CheckableUsersAdapter
+    private lateinit var usersRecyclerView: RecyclerView
+    private lateinit var progressbar: ProgressBar
+    private var selectedUsers: MutableList<ConnectycubeUser> = mutableListOf()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_create_chat)
+        initToolbar()
+        loadViews()
+        initUserAdapter()
+        initViews()
+        loadData()
+    }
+
+    private fun initToolbar() {
+        supportActionBar?.displayOptions = DISPLAY_SHOW_TITLE or DISPLAY_HOME_AS_UP
+        setTitle(R.string.new_chat)
+    }
+
+    private fun loadViews() {
+        usersRecyclerView = findViewById(R.id.list_users)
+        progressbar = findViewById(R.id.progressbar)
+    }
+
+    private fun initUserAdapter() {
+        usersAdapter = CheckableUsersAdapter(this, this)
+    }
+
+    private fun initViews() {
+        usersRecyclerView.layoutManager = LinearLayoutManager(this)
+        usersRecyclerView.itemAnimator = DefaultItemAnimator()
+        usersRecyclerView.adapter = usersAdapter
+    }
+
+    private fun loadData() {
+        createChatDialogViewModel = ViewModelProviders.of(
```

Comment:
consider to use inline function "viewModels" to create [ViewModel] like in other project components